db/migrate/20181215201843_add_auth_type_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthTypeToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :auth_type, :string, default: 'auth_code'
```

Comment:
I would put logic for making `auth_code` the default value in the model instead of in the migration, so the logic gets grouped all into one place.